src/styles/style-helpers.ts

```diff
@@ -25,6 +26,133 @@ function isTheme(args: ThemeArgs): args is Theme {
 export const getTheme = (args: ThemeArgs): Theme =>
   isTheme(args) ? args : args.theme;
 
+type Spacing = keyof Theme['spacings'] | null;
+
+const spacingString = (size: Spacing, theme: Theme) => {
+  if (!size) {
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn(
+        'A single `null` value was passed to the spacing mixin. This has no effect since `null` represents no set margin, not zero margin. If you are trying to reset the spacing, use custom styles instead.',
+      );
+    }
+    return null;
+  }
+  return css({ margin: theme.spacings[size] });
```

Comment:
Nothing happens. It would generate invalid styles (`margin:undefined;`) which are ignored by the browser. Best case, the developer realizes the mistake and fixes it, worst case, there's some dead code. 

The tradeoff here is performance. I hope that this mixin will be used abundantly, so I'd like to avoid any extra checks.

Reply:
Although, we could put these checks behind a `process.env.NODE_ENV !== 'production'` check, that would restrict the performance penalty to development 🤔

Reply:
Makes sense! So either we can add a check in development, but I think it's also fine like this (any good code editor should prevent a mistake like this from happening)